solr/core/src/java/org/apache/solr/core/CoreContainer.java

```diff
@@ -737,10 +746,10 @@ public void load() {
 
     zkSys.initZooKeeper(this, cfg.getCloudConfig());
     if (isZooKeeperAware()) {
-      pkiAuthenticationPlugin = new PKIAuthenticationPlugin(this, zkSys.getZkController().getNodeName(),
+      pkiAuthenticationSecurityBuilder = new PKIAuthenticationPlugin(this, zkSys.getZkController().getNodeName(),
           (PublicKeyHandler) containerHandlers.get(PublicKeyHandler.PATH));
       // use deprecated API for back-compat, remove in 9.0
-      pkiAuthenticationPlugin.initializeMetrics(solrMetricsContext, "/authentication/pki");
+      pkiAuthenticationSecurityBuilder.initializeMetrics(solrMetricsContext, "/authentication/pki");
```

Comment:
If this is for 9, is it possible to remove this?   Or is removing this mean other things need to happen that are orthogonal to this PR?

Reply:
Yeah, this is 9x, but I don't really see what that comment refers too - I don't see that initializeMetrics is deprecated in the super class chain or what this would be changed to.